docs/docs-new/pages/product/configuration/visualization-tools.mdx

```diff
@@ -99,6 +99,11 @@ providing a conversational interface for semantic layer:
     imageUrl="https://static.cube.dev/icons/delphi.svg"
     title="Delphi"
   />
+  <GridItem
```

Comment:
Should "Custom data experiences" become "AI- & LLM-based experiences" now? We have the same language on the [Integrations](https://cube.dev/integrations) page.

<img width="532" alt="Screenshot 2023-08-24 at 14 53 00" src="https://github.com/cube-js/cube/assets/3852894/1cf79f25-015d-4e9c-a88f-7bb4447ea30c">